src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/alerts-classic-vs-workflows.mdx

```diff
@@ -0,0 +1,912 @@
+      </td>
+      <td>
+        `$TARGETS`
+      </td>
+      <td>
+        string
+      </td>
+      <td>
+        List
+      </td>
+      <td>
+        `accumulations.targetName`
+      </td>
+      <td>
+        `[ { "id": "{{labels.targetId.[0]}}", "name": "{{#if accumulations.targetName}}{{accumulations.targetName.[0]}}{{else if entitiesData.entities}}{{entitiesData.entities.[0].name}}{{else}}N/A{{/if}}", "link": "{{issuePageUrl}}", "product": "{{accumulations.conditionProduct.[0]}}", "type": "{{#if entitiesData.types.[0]}}{{entitiesData.types.[0]}}{{else}}N/A{{/if}}", "labels": { {{#each accumulations.rawTag}}{{#if this.[0]}}"{{@key}}":{{json this.[0]}},{{/if}}{{/each}}
+]`
+      </td>
+      <td>
+        N/A
+      </td>
+    </tr>
+
+
+    <tr>
+      <td>
+        `timestamp`
+      </td>
+      <td>
+        `$TIMESTAMP`
+      </td>
+      <td>
+        number
+      </td>
+      <td>
+        number
+      </td>
+      <td>
+        `{{createdAt}}`
+      </td>
+      <td>
+        `{{#if closedAt}}{{closedAt}}{{else if acknowledgedAt}}{{acknowledgedAt}}{{else}}{{createdAt}}{{/if}}`
+      </td>
+      <td>
+        the timestamp of the issue creation
+      </td>
+    </tr>
+
+
+    <tr>
+      <td>
+        `timestamp_utc_string`
+      </td>
+      <td>
+        `$TIMESTAMP_UTC_STRING`
+      </td>
+      <td>
+        string
+      </td>
+      <td>
+        string
+      </td>
+      <td>
+        `{{issueCreatedAtUtc}}`
+      </td>
+      <td>
+        `{{#if issueClosedAtUtc}}{{json issueClosedAtUtc}}{{else if issueAcknowledgedAt}}{{json issueAcknowledgedAt}}{{else}}{{json issueCreatedAtUtc}}{{/if}}`
+      </td>
+      <td>
+        the timestamp of the issue creation
+      </td>
+    </tr>
+
+
+    <tr>
+      <td>
+        `version`
+      </td>
+      <td>
+        `$VERSION`
+      </td>
+      <td>
+        string
+      </td>
+      <td>
+        string
+      </td>
+      <td>
+        `N/A`
+      </td>
+      <td>
+        `"1.0"`
+      </td>
+      <td>
+        There is no matching attribute on the issue level.
+      </td>
+    </tr>
+
+
+    <tr>
+      <td>
+        `violation_callback_url`
+      </td>
+      <td>
+        `$VIOLATION_CALLBACK_URL`
+      </td>
+      <td>
+        string
+      </td>
+      <td>
+        string
+      </td>
+      <td>
+        `{{issuePageUrl}}`
+      </td>
+      <td>
+        `{{issuePageUrl}}`
+      </td>
+      <td>
+        N/A
+      </td>
+    </tr>
+
+
+    <tr>
+      <td>
+        `violation_chart_url`
+      </td>
+      <td>
+        `$VIOLATION_CHART_URL`
+      </td>
+      <td>
+        string
+      </td>
+      <td>
+        string
+      </td>
+      <td>
+        `{{violationChartUrl}}`
+      </td>
+      <td>
+        `{{violationChartUrl}}`
+      </td>
+      <td>
+        Link to the first violation chart in the issue
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+
+## Alerts (classic) default payload in workflows [#alerts-classic-payload]
```

Comment:
This section is basically the same as the previous, but in a different form.  What do you think about starting a CollapserGroup here and just having the Collapser for the Webhook template?  PagerDuty and others can be added later.